plugins/permissions/src/stickers/createGuildSticker.ts

```diff
@@ -0,0 +1,10 @@
+import { BotWithCache } from "../../deps.ts";
+import { requireBotGuildPermissions } from "../permissions.ts";
+
+export async function createGuildSticker(bot: BotWithCache) {
+  const createGuildSticker = bot.helpers.createGuildSticker;
+  bot.helpers.createGuildSticker = (guildId, options) => {
+    requireBotGuildPermissions(bot, guildId, ["MANAGE_EMOJIS"]);
```

Comment:
is this the right perm? not MANAGE_EMOJIS_AND_STICKERS?

Reply:
@Skillz4Killz #2473 haven't been merged yet

Reply:
Deno check is screaming because of BigString support, waiting for #2486 to merge before working on this